R/plots.R

```diff
@@ -155,7 +160,13 @@ plotStudy <- function(study, modelID, featureID, plotID, testID = NULL, librarie
     }
   }
 
-  returned <- f(plottingData)
+  if (dynamic == T){
+    returned <- plotly::plotly_json(f(plottingData))
```

Comment:
Please add the argument `jsonedit = FALSE`. Otherwise it tries to open the interactive list viewer, which would require the package [listviewer](https://cran.r-project.org/package=listviewer).